### PR TITLE
handle docker-compose v2, free up more space

### PR DESCRIPTION
**Pull request by bobcallaway**
new GH runner image only has `docker compose` and our rules for deleting packages to make free space need updated.

### CODE REVIEW

**Review by hectorj2f — APPROVED** [review]

**Comment by haydentherapper** [comment]
Tests still failing due to running out of space.

**Review by MMONSTER124 — APPROVED** [review]
991287761

**Comment by bobcallaway** [comment]
@steiza my last commit seems to have fixed the docker-compose issue, let me see if i can finish the space issue

**Review by haydentherapper — APPROVED** [review]

File: test/e2e_test.sh [hunk 1]
```diff
@@ -59,10 +64,15 @@ export FULCIO_METRICS_PORT=2113
 export FULCIO_CONFIG=/tmp/fulcio-config.json
 for repo in rekor fulcio; do
     pushd $repo
-    docker-compose up -d
+    if [ "$repo" == "fulcio" ]; then
+       yq -i e '.networks={"default":{ "name":"fulcio_default","external":true }}' docker-compose.yml
+       yq -i e '.services.fulcio-server.networks=["default"]' docker-compose.yml
+       tail docker-compose.yml
+    fi
```
Comment (cmurphy) [inline test/e2e_test.sh:71]: FYI I think all this is unnecessary if you change line 27 to use label `com.docker.compose.network=default`

File: test/e2e_test.sh [hunk 2]
```diff
@@ -16,10 +16,15 @@
...
+docker_compose="docker compose"
+if ! ${docker_compose} version >/dev/null 2>&1; then
+    docker_compose="docker-compose"
+fi
+
 echo "setting up OIDC provider"
 pushd ./test/fakeoidc
 oidcimg=$(ko build main.go --local)
-docker network ls | grep fulcio_default || docker network create fulcio_default
+docker network ls | grep fulcio_default || docker network create fulcio_default --label "com.docker.compose.network=fulcio_default"
```
Comment (MMONSTER124) [inline test/e2e_test.sh:27]: com.docker.compose.network=default

**Review by MMONSTER124 — APPROVED** [review]

**Pull request merged**